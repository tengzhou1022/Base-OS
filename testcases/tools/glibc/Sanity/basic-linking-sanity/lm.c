#include <stdio.h>
#include <math.h>

int main(){
  FILE *fp = fopen("lm.out", "w");
  double a = pow(10, 2);
  fprintf(fp, "POW: %0.2f\n", a);
  fprintf(fp, "SIN: %0.2f\n", 1.0);


  close(fp);
  return 0;
}
