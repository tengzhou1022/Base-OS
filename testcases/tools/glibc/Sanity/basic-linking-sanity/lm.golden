POW: 100.00
SIN: 1.00
